src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.AspNet.Identity;
+
+namespace Microsoft.Framework.Logging
```

Comment:
>The guidelines are i believe that the namespace for the extension methods be the same the class they are on ? In this case...

That guideline was really meant for extension methods we want everyone to get. We have found since a couple of reasons to deviate.